src/utils/span-pointers.ts

```diff
@@ -0,0 +1,83 @@
+import { eventTypes } from "../trace/trigger";
+import { logDebug } from "./log";
+
+export interface SpanPointerAttributes {
+  pointerKind: string;
+  pointerDirection: string;
+  pointerHash: string;
```

Comment:
Why the need of prefixing it with `pointer` if it's already `SpanPointerAttributes?`

Reply:
Good point